src/usr/local/www/services_router_advertisements.php

```diff
@@ -193,8 +193,8 @@
 			$input_errors[] = gettext("Maximum advertisement interval must be no less than 4 and no greater than 1800.");
 		}
 	}
-	if ($_POST['raadvdefaultlifetime'] && !is_numericint($_POST['raadvdefaultlifetime'])) {
-		$input_errors[] = gettext("Router lifetime must be an integer between 0 and 9000.");
+	if ($_POST['raadvdefaultlifetime'] && (($_POST['raadvdefaultlifetime'] < "1") || ($_POST['raadvdefaultlifetime'] > "9000"))) {
```

Comment:
Why are you putting integer values in quotes? If you are comparing numerically and want to force both to the same type, you should cast the strings to int rather than doing this.

Reply:
Also if the max is 3 * `ramaxrtradvinterval` why are you comparing against 9000 and not calculating the max based on `ramaxrtradvinterval`?

Reply:
quotes fixed.

3 * `ramaxrtradvinterval` is default, not maximum:
> Must be either zero or between MaxRtrAdvInterval and 9000 seconds.
> 
> Default: 3 * MaxRtrAdvInterval (Minimum 1 second).